server/feeders/reviewsgenerator.js

```diff
@@ -107,16 +109,24 @@ var triggerRestaurantsRatings = function() {
   var servicePath;
 
   var q = async.queue(function(task, callback) {
-
+    var restaurantId = utils.asciiEncode(
+      utils.stripForbiddenChars(
+        unescape(task.restaurantName)));
     setTimeout(function() {
-      utils.getListByType('Restaurant', task.restaurantName, fiwareHeaders)
+      utils.getListByType(
+        'Restaurant',
+        restaurantId,
+        fiwareHeaders)
       .then(function(data) {
         var fwHeaders = JSON.parse(JSON.stringify(fiwareHeaders));
         if (typeof data.body.department !== 'undefined') {
           fwHeaders['fiware-servicepath'] = '/' + data.body.department;
         }
-        utils.sendRequest('PATCH', task.aggregateRatings,
-          task.restaurantName, fwHeaders)
+        utils.sendRequest(
+          'PATCH',
```

Comment:
As we are generating reviews, we need to update the restaurant Ratings. So every time a review is added, it recalculates the rating and updates the field in the Restaurant entity.